specification/resources/registry/registries_create.yml

```
@@ -0,0 +1,51 @@
+operationId: registries_create
+
+summary: "[GA] Create Container Registry"
```

```suggestion
summary: "Create Container Registry"
```

If this API is generally available, it does not need to have a `[GA]` tag. That should be the default state. We only need to label endpoints that are not generally available so that customers understand that they may not have access.

---

> Is there a way we can tag the `/registry` endpoints without marking them as `deprecated`?

Short of marking them `deprecated`, the only way is updating the descriptions.

I'd suggest at least updating the top-level Container Registry description to include some information about the differences between the `/v2/registry` and `/v2/registries` endpoints:

https://github.com/digitalocean/openapi/blob/fe9078d917f5b311949397025701e9eab3e0901e/specification/DigitalOcean-public.v2.yaml#L132-L141